lib/matplotlib/cbook.py

```diff
@@ -566,6 +566,18 @@ def process(self, s, *args, **kwargs):
                     self._remove_proxy(proxy)
 
 
+class EventEmitter(object):
+    def __init__(self):
+        super(EventEmitter, self).__init__()
```

Comment:
Basically you need ``super`` for all classes that appear in the inheritance tree, not just because of what lies below, but what lies next to, it needs to call the next branch in the inheritance tree, what I would call a sibling class.  Having learnt about ``super`` only in the past couple of days, I now find it very beautiful and elegant and don't know how I have ever lived without it.

Do you think I should also add something about this in the developers guide?